### DISCUSSION TITLE
[A-L] (2023/24) Foglio 16 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#290] [Esercizi - Viaggi/Piazza]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/6059bd88-1732-4038-a8a7-1c8aceb83252)

**Comment by Elia-Belli** [comment]
Sviluppo con Laplace la seconda riga:
$$det(A)=(-1)^4\cdot (-1)\cdot det(A_{22})+(-1)^5\cdot 2\cdot det(A_{23})$$

$A_{22}$ e $A_{23}$ sono matrici $2\times 2$ quindi per calcolare il loro determinante utilizzo la definizione:
$$det(A)=-1\cdot (3\cdot 1-1\cdot 0)-2\cdot (3\cdot 1-1\cdot 1)=-3-4=-7$$

> Sto utilizzando la notazione $A_{ij}$ per il complemento algebrico dell'elemento $a_{ij}$ : nuova matrice senza riga $i$ e colonna $j$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/af868fd5-3101-48fb-b47b-886726b9bbb2)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-07 233954](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/c0c8e01c-c19d-4a8e-b10b-cc56d5ed6af2)

**Reply by daromacarlo** [reply]
![Immagine 2024-01-08 000450](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/4f349a39-c753-4354-87c9-9bc5a7c11bf2)
